<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageLight</class>
 <widget class="QWidget" name="PageLight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>126</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="layout">
   <item row="1" column="0">
    <widget class="QLabel" name="labelType">
     <property name="text">
      <string>&amp;Type</string>
     </property>
     <property name="buddy">
      <cstring>cbType</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cbType"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelDirection">
     <property name="text">
      <string>&amp;Direction</string>
     </property>
     <property name="buddy">
      <cstring>cbDirection</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cbDirection"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelDelay">
     <property name="text">
      <string>D&amp;elay</string>
     </property>
     <property name="buddy">
      <cstring>sliderDelay</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSlider" name="sliderDelay">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="pageStep">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelBrightness">
     <property name="text">
      <string>&amp;Brightness</string>
     </property>
     <property name="buddy">
      <cstring>sliderBrightness</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSlider" name="sliderBrightness">
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cbType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PageLight</receiver>
   <slot>onLightTypeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
